demo/nexmark/nexmark.sql

```diff
@@ -0,0 +1,60 @@
+CREATE TABLE person (
+    id BIGINT,
+    name VARCHAR,
+    emailAddress VARCHAR,
+    creditCard VARCHAR,
+    city VARCHAR,
+    state VARCHAR,
+    date_time TIMESTAMP(3), -- NOT NULL LATENESS INTERVAL 4 SECONDS,
```

Comment:
now we support lateness and watermarks for these columns, which could potentially affect performance significantly

Reply:
Do I just uncomment these?

Reply:
you can, but the semantics is slightly different.
for example, they will skip processing tuples that are too much out of order
so this may make the pipeline very fast

Reply:
The version I just pushed uses lateness by default, or you can specify `--no-lateness` to turn if off.